src/components/users/Profile.js

```diff
@@ -47,11 +48,15 @@ export default class Login extends Component {
       display: 'inline-block',
     };
 
+    var player = storeUtil.getPlayer(this.props.user.playerId);
+
     return (
       <Paper zDepth={1} style={paperStyle}>
         <h3>{this.context.t('profile.headerText')}</h3>
 
-        <p>{this.context.t('profile.loggedInText')}&nbsp;{this._reverseName((storeUtil.getPlayer(this.props.user.playerId) || {name: 'Broken Code'}).name)}</p>
+        <p>{this.context.t('profile.loggedInText')}&nbsp;{this._reverseName((player || {name: 'Broken Code'}).name)}</p>
+        <p>{this.context.t('profile.email')}&nbsp;{player.email}</p>
```

Comment:
Zoals bij de naam:

Iedereen weet zijn eigen adres, telefoonnummer en naam.
Het heeft geen enkele zin dit te tonen naar de gebruiker toe.

Wat wel op de "roadmap to v2" staat is het wijzigen van deze gegevens.
Dus volgend scenario: 
- Gebruiker gaat naar zijn profiel pagina
- Gebruiker ziet dat zijn email niet correct (meer) is
- Gebruiker corrigeert zijn email

Wat nu het geval is:
- Gebruiker komt op profiel
- Gebruiker ziet dat zijn email niet correct (meer) is
- Gebruiker is gefrustreerd